### ISSUE TITLE
How to do parameter update in grounded_sam_demo.py

### ISSUE THREAD

**Issue by 3D3C** [IDEA-Research/Grounded-Segment-Anything#398]
Hi author, in grounded_sam_demo.py there are only text prompts, I added a mask prompt so I need to retrain the parameters, but I still have no parameter update after removing the parameter freeze code in SamPredictor. So I would like to ask if grounded_sam_demo.py can do parameter updates or do I need to rewrite a fine-tuning code myself?

**Comment by rentainhe** [comment]
You can try to use the open-sourced Grounding-DINO training code as https://github.com/IDEA-Research/GroundingDINO/issues/241 to finetune grounding-dino on your own dataset and replace the original weights of G-DINO in Grounded-SAM to get a better result
